libraries/azure_vpn_gateways.rb

```diff
@@ -0,0 +1,46 @@
+require 'azure_generic_resources'
+
+class AzureVPNGateways < AzureGenericResources
+  name 'azure_vpn_gateways'
+  desc 'Verifies settings for AKS VPN Gateways'
+  example <<-EXAMPLE
+    azure_vpn_gateways(resource_group: 'example') do
+      it{ should exist }
+    end
+  EXAMPLE
+
+  attr_reader :table
+
+  def initialize(opts = {})
+    # Options should be Hash type. Otherwise Ruby will raise an error when we try to access the keys.
+    raise ArgumentError, 'Parameters must be provided in an Hash object.' unless opts.is_a?(Hash)
+
+    opts[:resource_provider] = specific_resource_constraint('Microsoft.Network/virtualNetworkGateways', opts)
+
+    # static_resource parameter must be true for setting the resource_provider in the backend.
+    super(opts, true)
+
+    # Check if the resource is failed.
+    # It is recommended to check that after every usage of inherited methods or making API calls.
+    return if failed_resource?
+
+    # Define the column and field names for FilterTable.
+    # In most cases, the `column` should be the pluralized form of the `field`.
+    # @see https://github.com/inspec/inspec/blob/master/docs/dev/filtertable-usage.md
+    table_schema = [
+      { column: :names, field: :name },
+      { column: :ids, field: :id },
+      { column: :types, field: :type },
+      { column: :locations, field: :location },
+      { column: :tags, field: :tag },
```

Comment:
This should be `{ column: :tags, field: :tags },`. Both `tags` are plural. See, [here](https://docs.microsoft.com/en-us/rest/api/network-gateway/virtualnetworkgateways/list#virtualnetworkgateway).

Reply:
I've amended this now, thanks.